modules/Footer.tsx

```diff
@@ -0,0 +1,16 @@
+import Image from 'next/image';
+export const Footer = () => {
+  return (
+    <footer className="flex justify-between items-center my-8">
+      <a className="block" href="https://vercel.com?utm_source=lyonjs&utm_campaign=oss">
+        <Image
+          alt="Vercel"
+          width="200"
+          height="100"
+          src="https://images.ctfassets.net/e5382hct74si/78Olo8EZRdUlcDUFQvnzG7/fa4cdb6dc04c40fceac194134788a0e2/1618983297-powered-by-vercel.svg"
```

Comment:
🤔 Can we have it locally instead ?
```suggestion
          src="/public/sponsor/powered-by-vercel.svg"
```

Reply:
We could use it for production too for SEO it would be really great 
RSS, Sitemap etc
I followed what is asked by vercel so I put the absolute URL.
Pretty sure they have a crawler to check it automatically

Reply:
Related to this doc https://vercel.com/guides/can-vercel-sponsor-my-open-source-project